data/json/items/melee.json

```diff
@@ -2333,5 +2333,24 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "type": "TOOL",
+    "category": "weapons",
+    "id": "grip_hook",
+    "name": "grip hook",
+    "description": "A simple steel meathook with a simple steel handle. Makes a passable melee weapon.",
+    "weight": 1400,
+    "to_hit": 1,
+    "color": "light_gray",
+    "symbol": "?",
+    "material": [ "steel" ],
+    "techniques": [ "DEF_DISARM", "SWEEP" ],
+    "flags": [ "DURABLE_MELEE", "BELT_CLIP" ],
+    "volume": 3,
+    "bashing": 11,
```

Comment:
What kind of grip hook are we picturing here? The ones I'm familiar with are relatively light and would be far more of a piercing than a bashing weapon...

Reply:
I notice that a "piercing" attribute is not used anywhere else in the codebase. I'm happy to bring bashing down, but how do I indicate that this thing is fairly good at piercing?

Reply:
It looks like presently piercing is just a monster thing, so pokey weapons have been defined as "cutting" in most of the json... maybe with a "STAB" attribute.

Reply:
I have made the griphook lighter, and made it bashing 2, cutting 6, and added the 'STAB' flag